dasharo-performance/cpu-temperature.robot

```diff
@@ -125,19 +125,37 @@ CPU Temperature Without Load (Ubuntu)
     Execute Command In Terminal    sensors-detect --auto
     ${timer}=    Convert To Integer    0
     @{temperature_list}=    Create List
+    ${max_temperature}=    Set Variable    0
+    ${min_temperature}=    Set Variable    999
```

Comment:
@tym2k1 is there any particular logic here to define max at 0 and min at 999?

Reply:
The `min_temperature` later uses the python [min()](https://www.w3schools.com/python/ref_func_min.asp) function to set the value to the lesser one between the current temperature and the currently stored `min_temperature`. The value parsed to the `min()` function can't be undefined so by setting it to a high number we ensure that it will get overwritten. I suppose this logic might skew the results if the CPU temperature reading will surpass 999°C but such reading is unlikely to happen.

The reverse happens for `max_temperature`.

[Relevant Code](https://github.com/3mdeb/meta-rte/pull/70/commits/c69d3ed9d8cf24211c8424fc6592d6f7adf63401)